setup.py

```diff
@@ -63,6 +63,7 @@
     author_email="kirkhamj@janelia.hhmi.org",
     py_modules=["versioneer"],
     packages=find_packages(exclude=["tests*"]),
+    package_data={"nanshe_workflow": ["dask_config.yaml"]},
```

Comment:
Need to make sure `dask_config.yaml` lands in the `conda` package and any typical Python install.